modules/local/pixelator/collect_metadata.nf

```diff
@@ -70,8 +65,7 @@ process PIXELATOR_COLLECT_METADATA {
         parameters: params
     ]
 
-    def nextflowJson = JsonOutput.toJson(metadata)
-
+    def nextflowJson = groovy.json.JsonOutput.toJson(metadata)
```

Comment:
need to collect version info from this module

Reply:
This is done here together with testing: https://github.com/nf-core/pixelator/pull/123